tests/unit/proxy_configuration/test_new_proxy_info.py

```diff
@@ -32,7 +31,7 @@ async def test_throws_on_invalid_new_url_function() -> None:
     with pytest.raises(ValueError) as exc:  # noqa: PT011
         await config.new_proxy_info(None, None, None)
 
-    assert isinstance(exc.value.__cause__, InvalidURL)
+    assert isinstance(exc.value.__cause__, ValueError)
```

Comment:
I believe that check can be removed - the `pytest.raises` three lines up ensures the same thing.